aws/sdk/benchmarks/s3-express/README.md

```diff
@@ -0,0 +1,38 @@
+# S3 Express Benchmark
```

Comment:
**Question**: Any reason this should not have just been made part of the existing `s3-throughput` benchmark test suite as a new type of test/input to that?

Reply:
Because benchmark scenarios were prescribed by the internal benchmark doc, fitting it into the existing s3 benchmark suite didn't work smoothly, unfortunately.